packages/smooth_app/lib/helpers/product_cards_helper.dart

```diff
@@ -15,15 +15,14 @@ String getProductBrands(Product product, AppLocalizations appLocalizations) {
   if (brands == null) {
     return appLocalizations.unknownBrand;
   } else {
-    return formatProductBrands(brands, appLocalizations);
+    return formatProductBrands(brands);
   }
 }
 
-/// Correctly format word separators between words (e.g. comma in English)
-String formatProductBrands(String brands, AppLocalizations appLocalizations) {
-  final String separator = appLocalizations.word_separator;
-  final String separatorChar =
-      RegExp.escape(appLocalizations.word_separator_char);
+/// Correctly format word separators between words.
```

Comment:
Just a general question about this change: If you change the separator, will this impact was is sent to the server?

Reply:
Technically this is not a change, it's a simplification as the 126 translations were identical for both labels.
Anyway, indeed it's sent to the server.